Backend/Backend/Services/NotificationService.cs

```diff
@@ -77,14 +94,14 @@ public async Task<bool> AddEpisodeNotification(Guid PodcastId,Episode episode,Ap
             //Loop through all of them
             foreach (PodcastFollow follow in podcastFollow)
             {
-                // TODO Add an Apropriete Picture for the Notification
+                // Add Cover Art for Notification Media
                 Notification notification = new Notification()
                 {
                     UserId = follow.UserId,
                     Title = "Podcast : " + episode.Podcast.Name,
                     Message = "New Episode added : " + episode.EpisodeName,
-                    Link = episode.Id.ToString(),
-                    Media = "https://png.pngtree.com/png-vector/20211018/ourmid/pngtree-simple-podcast-logo-design-png-image_3991612.png",
```

Comment:
Since there would be other types of notification aswell so that is why I am putting link but in output ill have Link as PodcastId so that its easy for frontend